sdk/ai/ai-projects/src/agents/streaming.ts

```diff
@@ -76,8 +80,43 @@ export async function submitToolOutputsToRunStreaming(
   runId: string,
   options: SubmitToolOutputsToRunParameters,
 ): Promise<AgentEventMessageStream> {
+  validateThreadId(threadId);
+  validateRunId(runId);
   options.body.stream = true;
 
   return processStream(context.path("/threads/{threadId}/runs/{runId}/submit_tool_outputs", threadId, runId)
     .post(options));
 }
+
+
+function validateCreateThreadAndRunBodyParam(options: CreateRunParameters| CreateThreadAndRunBodyParam): void {
+  if ('additional_messages' in options.body && options.body.additional_messages && options.body.additional_messages.some(value => !["user", "assistant"].includes(value.role))) {
+    throw new Error("Role must be either 'user' or 'assistant'");
+  }
```

Comment:
Could there be enum's

Reply:
I added a function in the inputValidation file using an enum